src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFileUtilities.cs

```diff
@@ -105,7 +105,13 @@ public static bool IsLockFileStillValid(DependencyGraphSpec dgSpec, PackagesLock
             var projectRuntimesKeys = project.RuntimeGraph.Runtimes.Select(r => r.Key).Where(k => k != null);
             var lockFileRuntimes = nuGetLockFile.Targets.Select(t => t.RuntimeIdentifier).Where(r => r != null).Distinct();
 
-            if (!projectRuntimesKeys.SequenceEqual(lockFileRuntimes))
+
+            //BUG: 8661 - Making it order both lists so we dont have this bug again with SequenceEquals
```

Comment:
I don't see how either this PR or this `if` statement is related to https://github.com/NuGet/Home/issues/8661. Am I missing something?

Reply:
I don't think the reference to an issue is necessary tbh.

Reply:
This was a copy paste error.  To take care of this - will just take it out.

Reply:
Removed.